src/Server/ServersManager/ProtocolServersManager.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <Server/ServersManager/IServersManager.h>
+#include <Server/TCPProtocolStackFactory.h>
+#include <Poco/Net/HTTPServerParams.h>
+
+namespace DB
+{
+
+class ProtocolServersManager : public IServersManager
+{
+public:
+    using IServersManager::IServersManager;
+
+    void createServers(
+        const Poco::Util::AbstractConfiguration & config,
+        IServer & server,
+        std::mutex & servers_lock,
+        Poco::ThreadPool & server_pool,
+        AsynchronousMetrics & async_metrics,
+        bool start_servers,
+        const ServerType & server_type) override;
+
+    using IServersManager::stopServers;
```

Comment:
I do not think we can do it here. If omitted, the function will not exist on ProtocolServersManager, because it has a virtual override with different parameters.